openstudiocore/src/osversion/test/VersionTranslator_GTest.cpp

```diff
@@ -228,100 +228,6 @@ TEST_F(OSVersionFixture, VersionTranslator_ExampleComponent_1_5) {
   testExampleComponent(1, 5);
 }
 
-void testAllDefaultObjects(int major, int minor){
-  // iterate through osversion subfolders
-  osversion::VersionTranslator translator;
-
-  // iterate through osversion subfolders
-  openstudio::path resources = resourcesPath() / toPath("osversion");
-  for (boost::filesystem::directory_iterator it(resources); it != boost::filesystem::directory_iterator(); ++it) {
-    if (boost::filesystem::is_directory(it->status())) {
-
-      QString stem = toQString(it->path().stem()).replace("_", ".");
-      VersionString vs(toString(stem));
-      if (vs.major() == major && vs.minor() == minor && vs.patch() == 3){

```

Comment:
No prob, just trying to figure out if there was more to this test than meets the eye.